package.json

```diff
@@ -45,7 +51,7 @@
     "prepare": "husky install && patch-package",
     "release": "np --no-release-draft",
     "pretest": "npm run lint",
-    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
+    "test": "npm run build && node --experimental-vm-modules node_modules/jest/bin/jest.js",
```

Comment:
related: https://github.com/stylelint/stylelint/issues/5291#issuecomment-1328717676

Reply:
Unfortunately, we cannot easily switch to another testing framework because of the dependency on [`jest-preset-stylelint`](https://github.com/stylelint/jest-preset-stylelint). 😓 

But when we finish the migration to ESM, it's valuable to consider switching.